src/SourceBuild/content/prep.sh

```diff
@@ -192,3 +197,14 @@ fi
 if [ "$downloadPrebuilts" == true ]; then
   DownloadArchive Prebuilts false $artifactsRid
 fi
+
+if [ "$keepBinaries" == false ]; then
+  git config --global --add safe.directory /vmr
```

Comment:
We should support build scenarios outside a git environment - this is currently the case.  It is with build options like --source-repository exist.

Reply:
This work might be better done in ./build.sh where there is support for building outside of a git environment. However, this would get tricky when trying to work with the binary-scanner... as explained in https://github.com/dotnet/source-build/issues/4087#issuecomment-1930933536, I would like to use the binary-scanner after this "cleaning" of the binaries occurs, but this is complicated when cleaning the binaries occurs as part of the build.

Alternatively, I could duplicate the ./build.sh logic for building outside of a git environment in prep. Or I could manually specify the directories to look for (i.e. src)

Reply:
The requirement is to be able to strip the binaries outside of the build.

Reply:
Defined paths to ignore so that binaries can be stripped outside of the build: https://github.com/dotnet/installer/pull/18541/files#diff-a1d21cbd7f180cea469d4fb634d555679ca532f21d903b2b5df311f6a051aa6aR183